### PR TITLE
Move tunic from Medieval Stuff mod

### PR DESCRIPTION
**Pull request by Rail-Runner**
Move tunic and recipes for existing sleeveless tunic from Medieval Stuff mod into the main game.

Other changes:
1. Remove unnecessary plural from of sleeveless tunic.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -10006,12 +10006,29 @@
         "flags" : ["VARSIZE"],
         "coverage" : 90,
         "material_thickness" : 1
+    },{
+        "type" : "ARMOR",
+        "id" : "tunic",
+        "name" : "tunic",
+        "weight" : 70,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: That weight doesn't look right. 70 grams?

Comment (chaosvolt) [inline data/json/items/armor.json]: Not sure where I got the weight from. Overlooked it, it seems. What's the weight for sleeveless tunics and rag tunics?

Comment (Darkyhard) [inline data/json/items/armor.json]: Sleeveless tunic weighs 60 grams. I'm not sure how much tunics actually weigh IRL.

**Comment by chaosvolt** [comment]
Self-assigned and removed. Any particular problems, other than the existing conservation-of-mass issue?

**Comment by Coolthulhu** [comment]
I sometimes mass assign stuff before checking if everything was fixed. This isn't a passive-aggressive "fix it now" or anything like that, it's just that it is more convenient for me to assign everything I'm going to be testing/reviewing. I can stop if it's a problem.

Yes, it's the conservation of mass thing. When in doubt, checking something with similar values can be enough. For example, t-shirt has 1 thickness, 2 volume and 110 weight, so tunic could be around `5/2*110`, say 250.

**Comment by chaosvolt** [comment]
Ah, no worries. Just caused minor confusion on my part, sorry.

And that might work, assuming one doesn't go for the literal route and focus on the weight of the rags need to make them. However, instead of that, 250 seems fine. Might be a possible reason to bump up the sleeveless tunic's weight.

**Comment by Coolthulhu** [comment]
Sleeveless tunic is significantly worse than the one with sleeves, it could use a boost.

**Pull request merged**